skore/src/skore/sklearn/cross_validate.py

```diff
@@ -14,16 +14,16 @@
 )
 from skore.project import Project
 
-
-def _find_ml_task(
-    estimator, y
-) -> Literal[
+MLTask = Literal[
```

Comment:
As this type is used outside of the module, should we start a `types.py` file in project's root ?

Reply:
Instead of project root, why not in the `sklearn/__init__.py`?

Reply:
This results in a circular import. I can create `sklearn/types.py`?

Reply:
done